config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
+  get '/', to: 'home#index'
+  get '/login', to: 'session#new'
+  post '/login', to: 'session#create'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.